cmd/gcp-controller-manager/node_csr_approver.go

```diff
@@ -860,9 +893,29 @@ func shouldDeleteNode(ctx *controllerContext, node *v1.Node, getInstance func(*c
 	}
 	// Instance with no alias range is route based, for which node object deletion is unnecessary.
 	klog.V(2).Infof("Instance %q has no alias range.", inst.Name)
+
 	return false, nil
 }
 
+func deleteAllPodsBoundToNode(ctx *controllerContext, nodeName string) error {
+	klog.V(1).Infof("start delete all pods pod on node %v", nodeName)
```

Comment:
Let's remove this log since many cases node will register for first time and there will no pods to delete so this will be redundant